src/ShowCommand.php

```diff
@@ -3,18 +3,21 @@
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Input\InputArgument;
 use GuzzleHttp\Client;
 
 class ShowCommand extends Command {
 
-    protected function configure()
-    {
+    const MAX_LENGTH = 130; //Largo maximo de caracteres para que no se rompa la tabla
```

Comment:
Please use comments when strictly neccesary. Try to reword the MAX_LENGHT naming to something that is clearer to the outcome of the value.

Also always code in English.